packages/cozy-client/src/store/documents.js

```diff
@@ -7,9 +7,15 @@ import isEqual from 'lodash/isEqual'
 const storeDocument = (state, document) => {
   const type = document._type
   if (!type) {
-    throw new Error('Document without _type', document)
+    if (process.env.NODE_ENV !== 'production') {
+      console.warn('Document without _type', document)
+    }
+    throw new Error('Document without _type')
   }
   if (!document._id) {
+    if (process.env.NODE_ENV !== 'production') {
+      console.warn('Document without _id', document)
+    }
     throw new Error('Document without _id', document)
```

Comment:
You want to delete document here ? like [here](https://github.com/cozy/cozy-client/pull/223/files#diff-1b915ff4945478dc0700c73893b71ae0R13)

Reply:
```suggestion
    throw new Error('Document without _id')
```